src/guide/deployment/mongodb.md

```diff
@@ -0,0 +1,155 @@
+# How to deploy a MongoDB
+
+We provide packages for the following versions of MongoDB:
+
+| Version | Store Path | Platform release |
+| --- | --- | --- |
+| 4.0.27 | `/nix/store/jg8mgbfkyg9vbc68dwizy7vj92gzzsvs-mongodb-4.0.27` | 2024_001/22.11 |
+| 4.2.24 | `/nix/store/86f2ff9w16whjmq3y0c6aywmxxd27mva-mongodb-4.2.24` | 2024_002/23.05 |
```

Comment:
I know these are kinda old, but I haven't seen any newer packages. Did I miss any?

(left out the 3.x stuff intentionally btw)

Reply:
Uhm … ich würde da eigentlich die Packages aus NixOS upstream nehmen wollen … https://search.nixos.org/packages?channel=23.11&show=mongodb&from=0&size=50&sort=relevance&type=packages&query=mongodb no?

Reply:
@zagy stimmt eigentlich :) 
Der Vollständigkeit halber würde ich die Tabelle noch beibehalten, aber mit ner Warnung versehen.

Reply:
Ah, ich weiß wieder, was das Problem war: mongodb ist unfree, d.h. man muss allowUnfree/allowUnfreePredicate setzen und das Ding selber kompilieren.

Reply:
@zagy mongodb_6 ist in den improtant packages ab dem kommenden 23.11.
Die anderen Storepfade für legacy Setups habe ich mal behalten.